content/docs/user-guide/interactivity/physics/nvidia-physx/best-practices.md

```diff
@@ -7,6 +7,6 @@ weight: 600
 See the following best practices when working with PhysX.
 + Colliders intersecting with terrain can result in unexpected behavior. For example, the object might rocket into space, jitter, or slow down performance. Avoid intersecting colliders with terrain. If you need to intersect a collider with terrain, use a small value for the collider size. These scenarios can be mitigated by clearing the **Persistent Contact Manifold** check box in the **Global Configuration** tab in the **PhysX Configuration** tool.
 + The **PhysX Character Controller component** must be on the same entity as the **Actor** component in order to work with the **Animation Editor**.
-+ If you select the **Static** check box in the **Transform** component for an entity that also has a **PhysX Rigid Body** component, the rigid body behaves statically and a warning appears about the incompatibility of the **PhysX Rigid Body** component and the **Static** transform option.
++ Only use **PhysX Static Rigid Body** when the **Static** check box in the **Transform** component of an entity is enabled. If you select the **Static** when there is also a **PhysX Dynamic Rigid Body** component, the rigid body behaves statically and a warning appears about the incompatibility of the **PhysX Dynamic Rigid Body** component and the **Static** transform option.
```

Comment:
nit: `is enabled.` -> `is checked.` 
`If you select the **Static** when there` -> `If you select **Static** when there` or `If you select the **Static** option when there`

Reply:
You're right this point could be better phrased. I'll split it 2 points and I'll rephrase it a little to make it more clear. Thanks!